src/ol/style/regularshape.js

```diff
@@ -401,11 +431,12 @@ ol.style.RegularShape.prototype.draw_ = function(renderOptions, context, x, y) {
   context.translate(x, y);
 
   context.beginPath();
+  var points = this.points_;
   if (this.radius2_ !== this.radius_) {
-    this.points_ = 2 * this.points_;
+    points = 2 * points;
```

Comment:
Yes, that is true, but for me it felt like a bug that getPoints would return the double amount of points i had given to the constructor. This isn't documented, too.